.appveyor-mingw.yml

```diff
@@ -0,0 +1,18 @@
+version: 1.0.{build}

```

Comment:
I don't have an issue with the MingW builds being a seperate yml file.  I guess I figured the PR would be running these builds.  I guess I'd have to set it up first it won't just detect that it needs those builds like travis would.